jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -88,6 +79,15 @@
       <unit id="org.slf4j.api" version="1.7.2.v20121108-1250"/>
     </location>
 
+    <!-- JBIDE-12815 JBoss Tools Locus bundles: not quite ready to be included -->
+    <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+        <repository location="http://repository.jboss.org/nexus/content/unzip/unzip/org/jboss/tools/locus/update.site/1.1.1.Final/update.site-1.1.1.Final.zip-unzip/"/>
+        <!-- JBIDE-12971 JBIDE-19110 Mockito
+        <unit id="org.jboss.tools.locus.mockito" version="1.9.5.Final_patched_TEIIDDES-1681-v20131024-0922-B58"/>  -->
```

Comment:
now mockito is *also* commented out.
My assumption is that this is a mistake and it was only fest-assert you wished to comment out.